var/spack/repos/builtin/packages/nmodl/package.py

```diff
@@ -11,10 +11,14 @@ class Nmodl(CMakePackage):
 
     homepage = "https://github.com/BlueBrain/nmodl.git"
     url      = "https://github.com/BlueBrain/nmodl.git"
+    git      = "https://github.com/BlueBrain/nmodl.git"
 
-    version('develop', branch='master', git=url, submodules=True, get_full_repo=True)
+    version('develop', branch='master', submodules=True, get_full_repo=True)
```

Comment:
Is this for version information?

Reply:
I thought the same. However I tried installing `nmodl` by cloning it for the first time or by using a local archive and it had the proper version, so I don't know why `get_full_repo=True` was used. I believe I can remove it. Also the revision is extracted by running `git log -1`, so we don't need anything more than that